packages/theme_tailor/lib/src/template/theme_extension_class_templates.dart

```diff
@@ -17,13 +17,19 @@ class ThemeExtensionClassTemplate {
       fieldsBuffer.write('final $value $key;');
     });
 
-    return '''
-    ${config.returnType}({
-      ${constructorBuffer.toString()}
-    });
+    if (config.fields.isEmpty) {
+      return '''
+      ${fieldsBuffer.toString()}
+      ''';
+    } else {
```

Comment:
We can do smth like this:
```dart
    if (config.fields.isEmpty) {
      return fieldsBuffer.toString();
    } else {
      return '''
      ${config.returnType}({
        $constructorBuffer
      });
    
      $fieldsBuffer
      ''';
    }
```
And there is no need to call to string on string buffers in string interpolation